client/app/docs/views/main.html

```diff
@@ -967,6 +967,13 @@ <h3 translate>Sample modal</h3>
                             scope.closeModal = function() {
                                 scope.isModalOpen = false;
                             };
+                            
+                            $(document).on('hidden.bs.modal', '.modal', function () {
+                                scope.modalActive = false;
+                                if (!scope.$$phase) {

```

Comment:
By the way, what is business logic doing here in the template? Since we are changing things here, I suggest we take this opportunity and refactor it out into an Angular component to avoid maintenance headaches in the future.